go.mod

```diff
@@ -19,21 +19,21 @@ require (
 	google.golang.org/protobuf v1.34.2
 	k8s.io/api v0.32.0-alpha.0
 	k8s.io/apimachinery v0.32.0-alpha.0
-	k8s.io/apiserver v0.31.0-rc.0
+	k8s.io/apiserver v0.31.0
 	k8s.io/client-go v1.5.2
```

Comment:
this should be 0.31.0

Reply:
I can still see `k8s.io/client-go v1.5.2`, it should be 0.31.0.

I know it's overridden below by `replace k8s.io/client-go => k8s.io/client-go v0.31.0`, so in the end the right version is used, but 1.5.2 might send an unaware person to a very wrong version of client-go.